editor.planx.uk/src/pages/FlowEditor/components/Sidebar/StyledTab.tsx

```diff
@@ -3,36 +3,23 @@ import Tab, { tabClasses, TabProps } from "@mui/material/Tab";
 import React from "react";
 import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 
-interface StyledTabProps extends TabProps {
-  tabTheme?: "light" | "dark";
-}
-
-const StyledTab = styled(({ tabTheme, ...props }: StyledTabProps) => (
+const StyledTab = styled((props: TabProps) => (
   <Tab {...props} disableFocusRipple disableTouchRipple disableRipple />
-))<StyledTabProps>(({ theme, tabTheme }) => ({
+))<TabProps>(({ theme }) => ({
   position: "relative",
   zIndex: 1,
   textTransform: "none",
   background: "transparent",
-  border: `1px solid transparent`,
   borderBottomColor: theme.palette.border.main,
-  color: theme.palette.primary.main,
-  fontWeight: FONT_WEIGHT_SEMI_BOLD,
+  color: theme.palette.text.primary,
+  minWidth: "75px",
   minHeight: "36px",
   margin: theme.spacing(0, 0.5),
-  marginBottom: "-1px",
-  padding: "0.5em",
+  padding: "0.75em 0.25em",
   [`&.${tabClasses.selected}`]: {
-    background:
-      tabTheme === "dark"
-        ? theme.palette.background.dark
-        : theme.palette.background.default,
-    borderColor: theme.palette.border.main,
-    borderBottomColor: theme.palette.common.white,
-    color:
-      tabTheme === "dark"
-        ? theme.palette.common.white
-        : theme.palette.text.primary,
+    fontWeight: FONT_WEIGHT_SEMI_BOLD,
+    color: theme.palette.text.primary,
+    boxShadow: `inset 0 -3px 0 ${theme.palette.warning.main}`,
```

Comment:
nit: not sure I like the use of the "warning" color for the underline now that we've been incorporating it for actual warnings (eg publish validation checks) - would the original blue be better / more neutral?

Reply:
Good shout, I was going for something to satisfies the following:
1.  Is somewhere visually between and contrasty to both black & white, so is visible as a marker for both dark & light tabs
2. Is not the primary action colour, as we use this to mark something that should be interacted with (rather than something that is "active")

The warning colour was an easy/lazy choice here (also leaning a bit too far into Github's tab styling 😀). I'll update to be the status agnostic "info" light blue colour that we use in the PlanX public interface, as this also satisfies the above.